packages/web3-wrapper/src/index.ts

```diff
@@ -1,2 +1,13 @@
 export { Web3Wrapper } from './web3_wrapper';
-export { Web3WrapperErrors, NodeType } from './types';
+export { marshaller } from './marshaller';
```

Comment:
Can you please add Javadoc comments to all the methods in `marshaller` now that it's public.